src/components/activities/Activity.jsx

```diff
@@ -82,51 +82,47 @@ const renderPoster = (posterUrl, activityName) => (
   />
 );
 
-const renderInfo = ({ id, location, name, price }) => (
+const renderInfo = ({ id, location, name, price, description }) => (
   <FullSizeCard fluid>
-    <MinHeightContent>
-      <Card.Header>{name}</Card.Header>
-    </MinHeightContent>
-    <Card.Content>
-      <Card.Description>
-        {location && (
-          <p>
-            <strong>Locatie: </strong>
-            <em>{location}</em>
-          </p>
-        )}
-        <p>
-          <strong>Prijs: </strong>
-          <em>
-            {price !== 0 ? (
-              <Currency quantity={parseFloat(price)} currency="EUR" />
-            ) : (
-              'Gratis!'
-            )}
-          </em>
-        </p>
-      </Card.Description>
-    </Card.Content>
-    <ButtonWrapper>
-      {/* without a div around the button the height of the label would not be taken into account */}
-      <Button
-        primary
-        href={'https://koala.svsticky.nl/activities/' + id}
-        target="_blank"
-        content="Inschrijven"
-        icon="external"
-        labelPosition="right"
-      />
-    </ButtonWrapper>
+    <FlexContainer>
+      <div className="title">
+        <h3>{name}</h3>
+      </div>
+      {location && (
+        <div className="location">
+          <strong>Locatie: </strong>
+          <em>{location}</em>
+        </div>
+      )}
+      <div className="price">
+        <strong>Prijs: </strong>
+        <em>
+          {price !== 0 ? (
+            <Currency quantity={parseFloat(price)} currency="EUR" />
+          ) : (
+            'Gratis!'
+          )}
+        </em>
+      </div>
+      <div className="description">
+        <em>{description}</em>
```

Comment:
I'm seeing some plain html in here (e.g. <a href="https://www.execut.nl/">https://www.execut.nl/</a>). Maybe we should wrap this in a `<Markdown>`?

Reply:
I chose to use [dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) as the data is not Markdown but HTML from Koala. This could be a risk, as it exposes the website to XSS. But, as we know where the HTML is coming from, and this should be secure, I don't think this would be a problem, but I am not 100% sure. Maybe @svsticky/it-crowd could also have a look. If we should sanitize the input, that would require a third-party dependency, which I try to avoid as much as possible...